src/coreComponents/constitutive/solid/DamageSpectralUtilities.hpp

```diff
@@ -33,7 +33,13 @@
 namespace geosx
 {
 
-//get the positive part only of tensor T using spectral split
+/**
+ *@brief Get the positive part only of a symmetric, 3x3, tensor T using spectral split. The eigenvectors and eigenvalues of T must be passed
+ * in.
+ *@param[in] eigs Array of eigenvalues of T.
+ *@param[in] eigevecs 3x3 array with the eigenvectors of T as rows.
+ *@param[out] positivePart Array that stores the positive part of T in Voigt Notation.
+ */
 GEOSX_HOST_DEVICE inline
 void PositivePartOfTensor( real64 (& eigs)[3], real64 (& eigvecs)[3][3], real64 (& positivePart)[6] )
```

Comment:
can you also make the function names lowercase?

Reply:
ok